rlclientlib/vw_model/safe_vw.cc

```diff
@@ -338,6 +346,19 @@ bool safe_vw::is_compatible(const std::string& args) const {
   return local_model_type == inbound_model_type;
 }
 
+bool safe_vw::is_CB_to_CCB_model_upgrade(const std::string& args) const
+{
+    const auto local_model_type = get_model_type(args);
```

Comment:
just to double check with comment at line 337.

Reply:
The initial values were incorrect. More details in the other comment.